speccy.js

```diff
@@ -51,4 +51,6 @@ program
 
 program.parse(process.argv);
 
-if (!program.args.length) program.help();
+if (!process.argv.slice(2).length) {
```

Comment:
Seems like this could throw index-out-of-range type error ...

Reply:
This snippet is taken from here https://github.com/tj/commander.js/issues/895#issuecomment-445430982

Reply:
Here's another thing https://teamtreehouse.com/community/why-we-need-processargvslice2

Reply:
Seems like slice() returns an empty array instead of an error